### DISCUSSION TITLE
v1.0

### DISCUSSION THREAD

**Discussion by rabbibotton** [rabbibotton/clog#77] [General]
This is the first release of CLOG and covers the base use of the web browser as a rendering engine locally or remotely as a GUI.

<hr /><em>This discussion was created from the release <a href='https://github.com/rabbibotton/clog/releases/tag/v1.0'>v1.0</a>.</em>

**Comment by aykaramba** [comment]
Thank you kindly Rabbi Botton.  Your work is greatly appreciated, looking forward to learning to use the toolkit better.
